CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Highlights
 
 * First instance modules
 
+* New standardised numeric predicates `NonZero`, `Positive`, `Negative`,
+  `NonPositive`, `NonNegative`, especially designed to work as instance
+  arguments.
```

Comment:
Something slightly strange seems to have happened to the CHANGELOG. These additions shouldn't be here. A merge mistake somewhere?

Reply:
There was a conflict in CHANGELOG.md, so I came up with a plan that did not work. I went into the master branch, and copied the latest version of the CHANGELOG.md, and then pasted those into my branch, and then modified my changes to fit in there.    I wasn't sure if I was supposed to resolve the merges myself, but I guess not since it did not work. So the strangeness is probably due to my strange attempt.

Reply:
I will revert the CHANGELOG, and try again.

Reply:
I reverted the CHANGELOG.md file to the state it was at when I forked the repository, and it seems to have resolved the conflict. Of course, I will have to make the changes again or else there will be no information on the changes I made in this pull request.   

Another guess I have for the cause of the conflict is that I may have changed the line   

* Added new proofs to ``Data.Sum.Properties`:

To have only one backtick between "to" and "Data". It seems erroneous to have two backticks there according to my understanding of Markdown syntax.